pkg/apis/networking/validation/validation_test.go

```diff
@@ -1227,6 +1247,25 @@ func TestValidateIngressTLS(t *testing.T) {
 			}
 		}
 	}
+
+	// Test for wildcard host and wildcard TLS
+	validCases := map[string]networking.Ingress{}
+	wildHost := "*.bar.com"
+	goodWildcardTLS := newValid()
+	goodWildcardTLS.Spec.Rules[0].Host = "*.bar.com"
+	goodWildcardTLS.Spec.TLS = []networking.IngressTLS{
+		{
+			Hosts: []string{wildHost},
+		},
+	}
+	goodWildcardTLSErr := fmt.Sprintf("spec.tls[0].hosts: Invalid value: '%v'", wildHost)
+	validCases[goodWildcardTLSErr] = goodWildcardTLS
```

Comment:
I'm not sure I understand the naming here. Aren't we expecting 0 errors as a result of this test case?

Reply:
Ya it's weird. But if I don't add this, a custom error is not thrown down below when we check to see if things are working. Is there some other way of naming this would increase clarity?